src/SDKs/CognitiveServices/data-plane/Language/Microsoft.CognitiveServices.Language.Tests/BaseTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.CognitiveServices.Language.TextAnalytics;
```

Comment:
@DavidLiCIG we have certain directory structure that is being used throughout the repo.
1) One being, if you have management and data plane the directories should be structured as
SDKs\CognitiveServices\management
SDKs\CognitiveServices\dataPlane
we rely on particular names to build and run tests during CI build.

Please change the directory structure accordingly.